app/controllers/api/v4/drugs_controller.rb

```diff
@@ -0,0 +1,50 @@
+class Api::V4::DrugsController < Api::V4::SyncController
+  skip_before_action :current_user_present?, only: [:sync_to_user]
+  skip_before_action :validate_sync_approval_status_allowed, only: [:sync_to_user]
+  skip_before_action :authenticate, only: [:sync_to_user]
+  skip_before_action :validate_facility, only: [:sync_to_user]
+  skip_before_action :validate_current_facility_belongs_to_users_facility_group, only: [:sync_to_user]
```

Comment:
Should we extract all this out into a "public API" mixin or something like that? Don't want facilities, protocols, and this drugs model to drift on this.